.github/workflows/ci.yml

```diff
@@ -6,15 +6,11 @@ jobs:
   static-analysis:
```

Comment:
I think you can simply remove the shell script and this static-analysis pipeline.

Reply:
resolved via [4733139](https://github.com/testcontainers/testcontainers-go/pull/569/commits/47331393f3cccb144105a759471f256a8ff820cf)